src/wallet/walletdb.cpp

```diff
@@ -569,6 +569,22 @@ bool HasLegacyRecords(CWallet& wallet, DatabaseBatch& batch)
     return false;
 }
 
+bool HasAnyRecordWithKeyPrefix(CWallet& wallet, const std::string& dbkey_prefix)
```

Comment:
I don't think this is really necessary, especially since it's test only.

Reply:
Ok, I can move it to the bench file.

Reply:
Could just remove it.